yaml/xpu_functions.yaml

```diff
@@ -300,3 +300,4 @@ supported:
   - nll_loss_forward
   - nll_loss_backward.grad_input
   - nll_loss_backward
+  - batch_norm_stats
```

Comment:
PR is titled to support operators, but only one seems to be exposed. Specifically, will `aten::native_batch_norm` be supported?

Reply:
Yes, batch_norm requires support for up to 12 ATen operators. This PR will enable them incrementally.